client/scripts/views/components/cw_button/cw_button.tsx

```diff
@@ -0,0 +1,29 @@
+/* @jsx m */
+
+import m from 'mithril';
+import './cw_button.scss';
+
+import { ComponentType, StyleProps } from '../component_kit/types';
+import { getClasses } from '../component_kit/helpers';
+
+export const CWButton: m.Component<
+  {
+    styleProps: StyleProps;
+    label: string;
+    onclick: (e?: MouseEvent) => void;
+  },
+  Record<string, unknown>
```

Comment:
can you explain this Record object as the state object?

Reply:
TS didn't like the empty object and recommended that instead. We should probably settle on an official type for Mithril component state if there isn't one already.